### PR TITLE
For the better understanding of time taken.

### PR DESCRIPTION
**Pull request by samyakpagariya**
### Describe your change:
In this change I have initialized a variable p with the value of (1e9+7) and then took the modulus of process time with it . This modification gives better time taken by the process . Firstly it was giving answer in the exponential now it gives in the integer form.

Fixed a small problem.

* [x] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/bubble_sort.py [hunk 1]
```diff
@@ -48,5 +48,6 @@ def bubble_sort(collection):
     user_input = input("Enter numbers separated by a comma:").strip()
     unsorted = [int(item) for item in user_input.split(",")]
     start = time.process_time()
+    p = (int)(1e9 + 7)
```
Comment (CaedenPH) [inline sorts/bubble_sort.py:51]:
```suggestion
```

File: sorts/bubble_sort.py [hunk 2]
```diff
@@ -48,5 +48,6 @@ def bubble_sort(collection):
...
     print(*bubble_sort(unsorted), sep=",")
-    print(f"Processing time: {time.process_time() - start}")
+    print(f"Processing time: {(time.process_time() - start)%p}")
```
Comment (CaedenPH) [inline sorts/bubble_sort.py:53]:
```suggestion
    print(f"Processing time: {(time.process_time() - start)%1e9 + 7}")
```

**Review by CaedenPH — APPROVED** [review]
@cclauss Looks good -

**Review by cclauss — APPROVED** [review]
RSLGTM

**Pull request merged**